packages/code-du-travail-frontend/public/static/modeles.css

```diff
@@ -0,0 +1,8 @@
+.courrier-expediteur {display: flex; align-items: flex-start; flex-direction:column;}
```

Comment:
Justement j'ai fait des branches pour ça  (pour régler l'erreur html sur ces pages): 
https://github.com/SocialGouv/code-du-travail-numerique/issues/3853#issuecomment-979311834

Reply:
Oui, mais justement, je les ai mis en global style. Car sinon, ça ne marche pas actuellement

Reply:
Pour moi, le style est mal géré. On devrait pas le gérer comme actuellement avec du css qui est présent en remote. On devrait le générer statiquement à la build.

Reply:
pourquoi ? moi je trouve ça pas mal d'avoir le css au même endroit que le html qui est généré

Reply:
Ah, tu parlais au niveau de la page via la gestion du module comme indiqué dans la documentation [https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css)

J'ai essayé, mais ça ne fonctionnait pas :(

Je comprends po pourquoi

Reply:
tu sais s'il va être minifié ce css ? Si c'est pas le cas on peut juste le committer minifié

Reply:
Alors, bonne nouvelle, j'ai vérifié et sur ma branche, on a les deux caractéristiques recherchés :

1. Le CSS n'est présent que sur la page qui utilise la classe dîtes, donc ça permet d'avoir le css qui se trimbale partout alors qu'il n'est qu'utile que sur une page
2. Le CSS que j'ai vu sur la page est bien minifié 🔥

Reply:
ok top